Builds/CMake/RippledCompiler.cmake

```diff
@@ -120,7 +120,7 @@ else ()
   target_link_libraries (common
     INTERFACE
       -rdynamic
-      $<$<BOOL:${is_linux}>:-Wl,-z,relro,-z,now>
+      $<$<BOOL:${is_linux}>:-Wl,-z,relro,-z,now,--build-id>
```

Comment:
Th unconditional addition of `--build-id` is intentional, but could be made conditional (on `voidstar`), like other options are. I just think it is a useful option to have even in a regular build.